### ISSUE TITLE
Links in docs style

### ISSUE THREAD

**Issue by olasitarska** [django/djangoproject.com#194]
One of the link styles in the docs is just a red color, no underlining. One of the HN users (thanks joseakle!) gave a feedback that this is not accessible for colorblind people.

Details: https://news.ycombinator.com/item?id=8758083
